setup.cfg

```
@@ -65,7 +65,7 @@ install_requires =
     ansimarkup>=1.0.0
     async-timeout>=3.0.0; python_version < "3.11"
     colorama>=0.4,<1
-    graphene>=2.1,<3
+    graphene==3.4.3
```

```suggestion
    graphene>=3.4.0
```

(see above comment in the conda file)

---

fixed (see above explanation for the version pin)